### PR TITLE
Prepare for 2.0.1 release

### LINKED ISSUE #17: updateheaders.py should end header with '# --' instead of '#--'
This is just a silly problem that causes a lot of fake output in the QA scripts

### PR DESCRIPTION
**Pull request by tovrstra**
This fixes most points in issue #68. It also fixes #17. (I was updating all headers anyway.)

### CODE REVIEW

**Comment by tovrstra** [comment]
Ready for review.

File: doc/tech_ref_release_history.rst [hunk 1]
```diff
@@ -22,6 +22,21 @@
 Release history
 ###############
 
+**June 17, 2016. Version 2.0.1**
+
+   - QA framework for automatically testing pull requests on Github with Travis-CI.
+   - Update config files for recent OSX, Ubuntu Linux and Fedora Linux versions.
+   - Fix: contractions of Gaussian functions are normalized when creating new basis sets.
+     (When loading wavefunctions from files that also contain a description of the basi
```
Comment (matt-chan) [inline doc/tech_ref_release_history.rst]: typo. "basis"

**Comment by matt-chan** [comment]
It all looks good other than the typo!

In the future, can we separate out PRs from auto-formatting changes (ie year, header, etc) and ones with code or hand-typed stuff? I can't make line comments on all the diffs because we had too many files in one PR. (No worries, the rest looked good on terminal).

**Comment by tovrstra** [comment]
Ah, good to know that large changes are causing troubles. I'll keep it in mind. I'll fix the typo. Thanks for catching it.

**Comment by matt-chan** [comment]
Merging!

**Pull request merged**